components/appbar/position.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Position
+page_title: AppBar Position
+description: Position settings of the AppBar for Blazor.
+slug: appbar-position
+tags: telerik,blazor,appbar,navbar,position
+published: True
+position: 35
+---
+
+# Position Settings
+
+This article outlines the available AppBar parameters, which control its position.
+
+>note Read the [CSS positioning MDN documentation article](https://developer.mozilla.org/en-US/docs/Web/CSS/position) to get a better understanding of how the AppBar component positioning works.
```

Comment:
I suggest removing this link. You should refer to such resources when there is something specific that you don't want to copy and paste here. Or when you need to point to a specific concept. 
This note sounds to me like "Go read this and then come back when you're smarter".